web/src/js/components/FlowView/Timing.tsx

```diff
@@ -71,51 +77,57 @@ export default function Timing({ flow }: { flow: Flow }) {
             deltaTo: ref,
         },
     ];
+
+    // Add HTTP-specific timestamps
```

Comment:
Remove unnecessary comment (I'll stop repeating this comment, please remove all comments that don't add any value)